packages/web/src/common/store/cache/store.test.ts

```diff
@@ -25,16 +25,38 @@ const initialConfirmerState = {
 }
 
 const MOCK_TIMESTAMP = 1479427200000
+vitest.mock('@audius/common/store', async () => ({
+  ...((await vitest.importActual('@audius/common/store')) as object),
```

Comment:
fun, do we need to spread or just return the result of importActual?

Reply:
I'm overriding `CACHE_PRUNE_MIN` below